src/recipes/ai-context/context.ts

```diff
@@ -2,21 +2,90 @@ import { promises as fs } from 'node:fs'
 import { dirname } from 'node:path'
 
 const ATTRIBUTES_REGEX = /(\S*)="([^\s"]*)"/gim
-const BASE_URL = 'https://docs.netlify.com/ai-context'
-export const FILE_NAME = 'netlify-development.mdc'
+const BASE_URL = process.env.AI_CONTEXT_BASE_URL ?? 'https://docs.netlify.com/ai-context'
+export const NTL_DEV_MCP_FILE_NAME = 'netlify-development.mdc'
 const MINIMUM_CLI_VERSION_HEADER = 'x-cli-min-ver'
 export const NETLIFY_PROVIDER = 'netlify'
 const PROVIDER_CONTEXT_REGEX = /<providercontext ([^>]*)>(.*)<\/providercontext>/ims
 const PROVIDER_CONTEXT_OVERRIDES_REGEX = /<providercontextoverrides([^>]*)>(.*)<\/providercontextoverrides>/ims
 const PROVIDER_CONTEXT_OVERRIDES_TAG = 'ProviderContextOverrides'
 
-export const downloadFile = async (cliVersion: string) => {
+export interface ContextConfig {
+  scope: string
+  glob?: string
+  shared?: string[]
+  endpoint?: string
+}
+
+export interface ContextFile {
+  key: string
+  config: ContextConfig
+  content: string
+}
+
+export interface ConsumerConfig {
+  key: string
+  presentedName: string
+  consumerProcessCmd?: string
+  path: string
+  ext: string
+  truncationLimit?: number
+  contextScopes: Record<string, ContextConfig>
+  hideFromCLI?: boolean
+  consumerTrigger?: string
+}
+
+let contextConsumers: ConsumerConfig[] = []
+export const getContextConsumers = async (cliVersion: string) => {
+  if (contextConsumers.length > 0) {
+    return contextConsumers
+  }
+  const res = await fetch(`${BASE_URL}/context-consumers`, {
+    headers: {
+      'user-agent': `NetlifyCLI ${cliVersion}`,
+    },
+  })
+
+  if (!res.ok) {
+    return []
+  }
+
   try {
-    const res = await fetch(`${BASE_URL}/${FILE_NAME}`, {
+    const data = (await res.json()) as { consumers: ConsumerConfig[] } | undefined
+    contextConsumers = data?.consumers ?? []
+  } catch {
+    return []
```

Comment:
nit: No need for an another exit point.

```suggestion
    // no-op
```